resources/basic/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.24)
```

Comment:
with these changes, I think it is time to put all of this under a ``tests`` folder of the project.